blog/static/js/main.js

```diff
@@ -0,0 +1,110 @@
+document.getElementById("titlelink")
+
+let element = document.getElementById("titlelink");
+
+element.style["color"] = "aqua";
+element.style["border"] = "3px solid aqua";
+
+function doTheStuff() { //defining a python function
+  let elements_posttitles = document.getElementsByClassName("posttitle");
+
+for (element of elements_posttitles){
+  element.style["border"] = "3px solid green";
+  
+  }
+}
+
+let theButton = document.getElementById("thebutton");
+theButton.addEventListener("click", doTheStuff);
+
+var flyingMen = [];
```

Comment:
Could you add a comment linking to where this code came from? Usually, people add something like 'based on code from ...'

Reply:
Also, could you add a comment explaining what this section of code does?